packages/nestjs-test-utilities/src/lib/builders/module/module.builder.ts

```diff
@@ -1,18 +1,16 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import {
-  DynamicModule,
-  ForwardReference,
-  Provider,
   Type,
+  Provider,
+  DynamicModule,
+  NotImplementedException,
 } from "@nestjs/common";
 import { Test, TestingModuleBuilder } from "@nestjs/testing";
 import * as _ from "lodash";
 
 export type NestJSModule =
   | Type<unknown>
   | DynamicModule
-  | Promise<DynamicModule>
```

Comment:
This might have some unforseen consiquenses, better to set m: any in the section below

Reply:
reverted